guides/common/modules/con_supported-usage-and-versions-of-project-components.adoc

```diff
@@ -0,0 +1,5 @@
+[id="Supported-Usage-and-Versions-of-Project-Components_{context}"]
+= Supported Usage and Versions of {Project} Components
+
+[role="_abstract"]
+Sections below provide details about which scenarios and versions are supported within {Project}.
```

Comment:
This is self-referential and not user-focused. I would consider something like "Find out which use cases, architectures, and versions are supported by {Project}."